plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/editparts/AbstractFBNElementEditPart.java

```diff
@@ -460,8 +474,18 @@ protected List<Object> getModelChildren() {
 				.filter(it -> !it.isVisible()).toList();
 		final List<VarDeclaration> outputRemovalList = getModel().getInterface().getOutputVars().stream()
 				.filter(it -> !it.isVisible()).toList();
+
+		final List<VarDeclaration> inoutInConList = getRemovedInOutConnections(true);
+		final List<VarDeclaration> inoutOutConList = getRemovedInOutConnections(false);
+
+		inoutInConList.stream().forEach(vd -> vd.setValue(null));
```

Comment:
add this to the method getRemovedInOutConnections. 
Furhtermore please rename getRemovedInOutConnections
it does not return connections, therefore it should be named getVarInOutPins